integration-tests/src/test/java/oracle/kubernetes/operator/ItManagedCoherence.java

```diff
@@ -373,4 +405,62 @@ private ExecResult clearCache() throws Exception {
     return result;
   }
 
+  private void restartCluster(String domainUid) throws Exception {
+    String podName = domainUid + "-new-managed-server1";
+
+    String restartTmpDir = getResultDir() + "/restarttemp";
+    Files.createDirectories(Paths.get(restartTmpDir));
+    
+    LoggerHelper.getLocal().log(Level.INFO, "created restart temp dir");
+
+    String originalYaml =
+            getUserProjectsDir()
+                + "/weblogic-domains/"
+                + domain.getDomainUid()
+                + "/domain.yaml";
+    
+    // Modify the original domain yaml to include restartVersion in admin server node
+    DomainCrd crd = new DomainCrd(originalYaml);
+    Map<String, Object> cluster = new HashMap();
+    cluster.put("restartVersion", "v1.1");
+    crd.addObjectNodeToCluster("dataCluster", cluster);
+    String modYaml = crd.getYamlTree();
+    LoggerHelper.getLocal().log(Level.INFO, modYaml);
+
+    // Write the modified yaml to a new file
+    Path path = Paths.get(restartTmpDir, "restart.cluster.yaml");
+    LoggerHelper.getLocal().log(Level.INFO, "Path of the modified domain.yaml :{0}", path.toString());
+    Charset charset = StandardCharsets.UTF_8;
+    Files.write(path, modYaml.getBytes(charset));
+
+    // Apply the new yaml to update the domain crd
+    LoggerHelper.getLocal().log(Level.INFO, "kubectl apply -f {0}", path.toString());
+    ExecResult exec = TestUtils.exec("kubectl apply -f " + path.toString());
+    LoggerHelper.getLocal().log(Level.INFO, exec.stdout());
+    LoggerHelper.getLocal().log(Level.INFO, "Verifying if the cluster is restarted");
+
+    TestUtils.checkPodReady(domainUid + "-new-managed-server1", domainNS1);
+    TestUtils.checkPodReady(domainUid + "-new-managed-server2", domainNS1);
+
+  }
+
+  private void verifyServersIngressRunning() throws Exception {
+    LoggerHelper.getLocal().log(Level.INFO, "Verifying server status");
+    domain.verifyServersReady();
+    LoggerHelper.getLocal().log(Level.INFO, "Verifying traefik and ingress created for the domain");
+    LoggerHelper.getLocal().log(Level.INFO,
+        TestUtils.exec("helm status traefik-operator").stdout(), true);
+    LoggerHelper.getLocal().log(Level.INFO,
```

Comment:
feels like we should be checking the returned values in this block of code, not just printing them out

Reply:
Does TestUtils.exec check the return value?  I agree that it's not obvious how the test code handles a failed helm command.

Reply:
This method was added while I was debugging the test failure initially, later Bhavani took over the branch. It was not meant to get it merged as is.

Reply:
I left it there thinking that more debug is good. Was going to delete this when I edit this file to remove the debug flags after we finally resolve the issue. I will remove it now.

Reply:
Already checking the status of all the servers before trying to deploy the app.